packages/runtime/test/src/runtime.ts

```diff
@@ -69,3 +70,76 @@ describe("Logic Signature Transaction in Runtime", function () {
     );
   });
 });
+
+describe("Rounds Test", function () {
+  useFixture("basic-teal");
+  let john = new StoreAccount(minBalance);
+  let bob = new StoreAccount(minBalance);
+  let runtime: Runtime;
+  let program: string;
+  let txnParams: AlgoTransferParam;
+  this.beforeAll(function () {
+    runtime = new Runtime([john, bob]); // setup test
+    program = getProgram('basic.teal');
+
+    // set up transaction paramenters
+    txnParams = {
+      type: TransactionType.TransferAlgo, // payment
+      sign: SignType.SecretKey,
+      fromAccount: john.account,
+      toAccountAddr: bob.address,
+      amountMicroAlgos: 100,
+      payFlags: { firstValid: 5, validRounds: 200 }
+    };
+  });
+
+  afterEach(function () {
+    john = new StoreAccount(minBalance);
+    bob = new StoreAccount(minBalance);
+    runtime = new Runtime([john, bob]);
+    txnParams.fromAccount = john.account;
+    txnParams.toAccountAddr = bob.address;
+  });
+
+  function syncAccounts (): void {
+    john = runtime.getAccount(john.address);
+    bob = runtime.getAccount(bob.address);
+  }
+
+  it("should pass if first and last valid are in current round range", () => {
+    txnParams.payFlags = { totalFee: 1000, firstValid: 5, validRounds: 200 };
+    // set round
+    runtime.setRound(20);
+
+    // execute transaction
```

Comment:
same here (also in other unit tests as well).

Reply:
ok